README.md

```diff
@@ -111,7 +113,86 @@ end
 ***Note: please adjust migration to yours Rails version***
 
 
-### Gladly Service side
+### Gladly Service side:
+
+Provide to your agent:
+- lookup endpoint (  `https://example-spree-store.com/api/v1/customers/lookup` ), where `https://example-spree-store.com` is **your** Spree store URL.
+- signing_key 
+
+
+## Events
+
+### Description
+Within gem we introducing [Conversations (Create Item)](https://developer.gladly.com/rest/#operation/createItem) using API client. 
+We implemented two events against `Spree::Order` model:
+ - `Placed` - it's fire up after Order is completed by customer ( `Gladly::Events::Order::Placed`)
+ - `Refundned` - it's fire up after Order items are returned to customer  ( `Gladly::Events::Order::Refunded`)
+ 
+More about [Conversations](https://developer.gladly.com/rest/#tag/Conversations)
+
+### Configuration
+
+```ruby
+config.gladly_api_username = 'api_username@example.com'
+config.gladly_api_key = 'api_key'
+config.gladly_api_base_url = 'https://dev-example.gladly.qa'
```

Comment:
I think we should mention explicitly, that if this is not set, events won't be sent

Reply:
I think there should also be another flag, that would keep the client functional but turn off the built-in events.
There may be cases where someone will want to implement their custom events, without sending OrderPlaced, OrderRefunded

Reply:
@rafalcymerys both issues addressed by this commit https://github.com/upsidelab/spree_gladly/pull/65/commits/78e8b0fe11f41fbb4c1845ece5a0baac7325e214